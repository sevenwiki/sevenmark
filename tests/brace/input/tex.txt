{{{#tex #block
E = mc^2
}}}