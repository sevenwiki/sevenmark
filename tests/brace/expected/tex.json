[
  {
    "TeXElement": {
      "location": {
        "start": 0,
        "end": 27
      },
      "is_block": true,
      "content": "E = mc^2\n"
    }
  }
]